db/migrations/20180117153013_backfill_movies_name.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.raw('UPDATE movies SET name = title');
```

Comment:
I think the miyagi documentation needs to be updated for this. You need to modify the query such that it only sets it for columns where name is `null`, otherwise you could end up overwriting a row that already has a value for name with `null`

Reply:
should be fixed now! ty